docs/devguide/create-module.asciidoc

```diff
@@ -77,6 +77,66 @@ the metricsets. This file does not have `_test.go` in the name because otherwise
 
 To see an example of the `testing.go` file, look at the https://github.com/elastic/beats/tree/master/metricbeat/module/mysql[mysql module].
 
+[float]
+===== Test a Metricbeat module manually
+
+To test a Metricbeat module manually, follow the steps below.
+
+First we have to build the docker image which is available for the modules. As an example let's take MySQL module.
+
+This steps assume you have checked out the Beats repository from Github and are inside `beats` directory:
+
+```
+cd metricbeat/module/mysql/_meta/
+docker build .
+```
+
+This changes your directory to where the Dockerfile is for MySQL and builds the docker image. In the end you should see something like:
+
+```
+...
+Removing intermediate container 0e58cfb7b197
+ ---> 9492074840ea
+Step 5/5 : COPY test.cnf /etc/mysql/conf.d/test.cnf
+ ---> 002969e1d810
+Successfully built 002969e1d810
+```
+
+The last has after `Successfully built` is the important one. It's the one we run our container on. Before we run the container we also need to know which port to expose. The port is listed in the `env` file:
```

Comment:
_The last Docker image ID after_... It's the Docker image we run our..._ Maybe it's worth adding `-t` when building the image so later the user is not using ID's but instead some `MetricbeatMySQL`